astropy/coordinates/earth.py

```diff
@@ -29,7 +29,12 @@
     "EarthLocation",
 ]
 
-GeodeticLocation = collections.namedtuple("GeodeticLocation", ["lon", "lat", "height"])
+
+class GeodeticLocation(NamedTuple):
+    lon: Longitude
+    lat: Latitude
+    height: u.Quantity
```

Comment:
Probably worth adding docstrings?

Reply:
Given discussion in https://github.com/astropy/astropy/pull/15276#discussion_r1315182860, we will probably add docstrings.

Reply:
This one does make sense to me (in a separate PR ;-), and is a good example of why inheriting from `NamedTuple` is a better idea than calling `namedtuple`. A class docstring may be most appropriate.